gammapy/datasets/tests/test_io.py

```diff
@@ -59,3 +64,9 @@ def test_datasets_to_io(tmp_path):
     assert dataset0.psf is not None
     assert dataset0.edisp is not None
     assert_allclose(dataset0.background_model.evaluate().data.sum(), 4094.2, atol=0.1)
+
+    Fit(datasets).run()
```

Comment:
I think this now requires a `@requires_minuit`.

Reply:
added @requires_dependency("iminuit")